xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 10.7.0
+JSONRPC_VERSION 10.7.1
```

Comment:
Changes to parameter names is a breaking change, so should be a bump to 11.0.0
The wrong names I have only been in the API for a matter of hours, but since it would have been better to bump version when we branched for v19 (and then could have picked which API issues were backported to v18) we may as well do it here.